src/olympia/devhub/templates/devhub/includes/addon_details.html

```diff
@@ -92,22 +92,24 @@
       {{ addon.last_updated|datetime(_('%%b %%e, %%Y')) }}
   </li>
 {% endif %}
-{% set latest_listed_version=addon.find_latest_version(channel=amo.RELEASE_CHANNEL_LISTED) %}
-{% if latest_listed_version and latest_listed_version != addon.current_version %}
-  <li>
-    <strong>{{ _('Next Version:') }}</strong>
-    {{ link_if_listed_else_text(latest_listed_version, latest_listed_version.version) }}
-    <span class="tip tooltip" title="{{ _('This is the newest uploaded version, however it isn’t live on the site yet.') }}">?</span>
-  </li>
-{% endif %}
-{% with position = get_position(addon) %}
-  {% if position and position.pos and position.total %}
-    <li class="queue-position" title="{{ _('Queues are not reviewed strictly in order') }}">
-      <strong>{{ _('Queue Position:') }}</strong>
-      {% trans position=position.pos|numberfmt,
-               total=position.total|numberfmt %}
-        {{ position }} of {{ total }}
-      {% endtrans %}
+{% if addon.has_listed_versions() %}
```

Comment:
we need something like `{% set version = item.addon.current_version or item.addon.find_latest_version(channel=amo.RELEASE_CHANNEL_UNLISTED) %}` below too or we won't be showing any version for unlisted add-ons.  (The box is getting an overhaul in #3471 and I have some questions in https://github.com/mozilla/addons/issues/193#issuecomment-259684821 but for now we should avoid removing functionality)